recipes/reef3d/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+make -j "${CPU_COUNT}"
```

Comment:
Sew above

Reply:
In this case, we need `mpicxx`, which is called out in the upstream Makefile. But I've put `CXX=mpicxx` to be more explicit. I can't find whether there is a variable for this available.